h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -463,6 +464,11 @@ public void init(boolean expensive) {
         case gaussian:
 //          if (_nclass != 1) error("_family", H2O.technote(2, "Gaussian requires the response to be numeric."));
           break;
+        case fractionalbinomial:
```

Comment:
**Question:** How should NaNs in response columns be treated ? The same as binomial ?

Reply:
Yes